bin/sozo/src/ops/register.rs

```diff
@@ -1,21 +1,47 @@
+use std::collections::HashMap;
+
 use anyhow::{Context, Result};
 use dojo_world::contracts::WorldContract;
 use dojo_world::metadata::Environment;
 use dojo_world::utils::TransactionWaiter;
+use scarb::core::Config;
 use starknet::accounts::Account;
+use starknet_crypto::FieldElement;
 
 use crate::commands::register::RegisterCommand;
 
-pub async fn execute(command: RegisterCommand, env_metadata: Option<Environment>) -> Result<()> {
+pub async fn execute(
+    command: RegisterCommand,
+    env_metadata: Option<Environment>,
+    config: &Config,
+) -> Result<()> {
     match command {
         RegisterCommand::Model { models, world, starknet, account, transaction } => {
             let world_address = world.address(env_metadata.as_ref())?;
             let provider = starknet.provider(env_metadata.as_ref())?;
 
             let account = account.account(&provider, env_metadata.as_ref()).await?;
             let world = WorldContract::new(world_address, &account);
+            let mut models_to_register = Vec::new();
+            for model_class_hash in models.iter() {
+                let (class_hash, _contract_address) =
+                    world.model(model_class_hash.into()).call().await?;
+                if class_hash.0 == FieldElement::ZERO {
+                    models_to_register.push(model_class_hash.clone());
+                } else {
+                    config.ui().print(format!(
+                        "\"{:?}\" model already registered with the given class hash",
```

Comment:
```suggestion
                        "\"{:#x}\" model already registered with the given class hash",
```

We want to print out the hex value of the class hash, instead of the debug.